lib/src/Pos/Diffusion/Subscription/Dns.hs

```diff
@@ -73,20 +73,20 @@ dnsSubscriptionWorker oq networkCfg DnsDomains{..} retryInterval sendActions = d
         dnsPeersList <- findDnsPeers index alts
         modifySharedAtomic dnsPeersVar $ \dnsPeers -> do
             let dnsPeers' = M.insert index dnsPeersList dnsPeers
-            void $ OQ.updatePeersBucket oq BucketBehindNatWorker $ \_ ->
+            void $ updatePeersBucket BucketBehindNatWorker $ \_ ->
                 peersFromList mempty ((,) NodeRelay <$> M.elems dnsPeers')
             pure (dnsPeers', ())
         -- Try to subscribe to some peer.
         -- If they all fail, wait a while before trying again.
         subscribeToOne dnsPeersList
-        retryInterval >>= delay
+        dtDuration keepAliveTimer >>= delay
```

Comment:
You don't have to do it for this pull request, but I think eventually we'll want this to be different from the keep alive timer. No sense waiting a whole slot duration without any connection before trying again, I think.